### ISSUE TITLE
NoMethodError: undefined method `strptime' for Time:Class

### ISSUE THREAD

**Issue by dankeller** [backup/backup#727]
I get this error when my backup enters the Cycling phase:
```
[2015/12/14 20:31:29][info] Performing Backup for [REDACTED]!
[2015/12/14 20:31:29][info] [ backup 4.2.2 : ruby 2.1.5p273 (2014-11-13) [x86_64-linux-gnu] ]
...
[2015/12/14 20:31:39][error] Model::Error: Backup for [REDACTED] Failed!
[2015/12/14 20:31:39][error] --- Wrapped Exception ---
[2015/12/14 20:31:39][error] NoMethodError: undefined method `strftime' for Time:Class
[2015/12/14 20:31:39][error]
[2015/12/14 20:31:39][error] Backtrace:
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/package.rb:51:in `time_as_object'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/storage/cycler.rb:21:in `block in cycle!'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/storage/cycler.rb:20:in `select'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/storage/cycler.rb:20:in `cycle!'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/storage/base.rb:47:in `perform!'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:334:in `block in store!'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:332:in `map'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:332:in `store!'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:306:in `block in procedures'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:269:in `call'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:269:in `block in perform!'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:268:in `each'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/model.rb:268:in `perform!'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/lib/backup/cli.rb:156:in `perform'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/thor-0.18.1/lib/thor/command.rb:27:in `run'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/thor-0.18.1/lib/thor/invocation.rb:120:in `invoke_command'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/thor-0.18.1/lib/thor.rb:363:in `dispatch'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/thor-0.18.1/lib/thor/base.rb:439:in `start'
[2015/12/14 20:31:39][error]   /var/lib/gems/2.1.0/gems/backup-4.2.2/bin/backup:5:in `<top (required)>'
[2015/12/14 20:31:39][error]   /usr/local/bin/backup:23:in `load'
[2015/12/14 20:31:39][error]   /usr/local/bin/backup:23:in `<main>'
```
Running Debian Jessie. Fresh install of backup gem and dependencies. Anyone else getting this?

**Comment by tombruijn** [comment]
Looks like something is messed up in the gem @dankeller . Looking into it. For now you can add `require 'time'` in your config file or model file at the top and it should work.

**Comment by dankeller** [comment]
Thanks for the workaround

**Comment by tombruijn** [comment]
Fixed in #729 
Will try to release soon
